pkg/arvo/sys/lull.hoon

```diff
@@ -2754,6 +2754,7 @@
             sky=(map spur path-state)
             ken=(jug spar:ames wire)
     ==  ==
+  +$  egg-any  $%([%15 egg])
```

Comment:
Can you add a comment to `$spore` in gall, noting that changes to its version tag need to be replicated here?